src/services/pocket-relayer.ts

```diff
@@ -408,6 +408,9 @@ export class PocketRelayer {
     )
 
     if (pocketSession instanceof Session) {
+      let syncCheckPromise: Promise<Node[]>
+      let chainCheckPromise: Promise<Node[]>
```

Comment:
these two variable names seem to be reduntant, could be changed to `syncCheck` and `chainCheck`. But that's up to you tho